### DISCUSSION TITLE
Compartment calling

### DISCUSSION THREAD

**Discussion by wsnoble** [open2c/cooler#265] [Q&A] [answered]
Am I correct in thinking that cooltools doesn't include any compartment calling functionality? Anyone got a recommendation for what compartment caller to use on a .cool file?

**Comment by nvictus** [comment, marked as answer]
UPDATE: today, these tools are exposed as [cooltools eigs-cis](https://cooltools.readthedocs.io/en/latest/cli.html#cooltools-eigs-cis) and [cooltools eigs-trans](https://cooltools.readthedocs.io/en/latest/cli.html#cooltools-eigs-trans)

---

[cooltools](https://cooltools.readthedocs.io/en/latest/cli.html#cooltools-call-compartments) has two eigenvector calculation tools -- one for cis interaction data and one for trans interaction data -- exposed to the command line as `cooltools call-compartments`. 

A bit of a misnomer in that it doesn't really "call" them in the sense of thresholding and assigning A vs B labels, but it will orient the eigenvectors to correlate positively with a provided phasing track (typically GC content).
